env/env_posix.cc

```diff
@@ -152,20 +152,12 @@ class PosixEnv : public Env {
   virtual Status NewSequentialFile(const std::string& fname,
                                    unique_ptr<SequentialFile>* result,
                                    const EnvOptions& options) override {
-    result->reset();
+    // force not using direct io for sequential file
```

Comment:
Does that mean that we only disable it for POSIX, also suppressing the options that I am passing to the Env sounds to me like something that will confuse future developers.

Maybe it's better to construct the optimized options outside the Env and then pass it.
What do you think ?